### PR TITLE
Feat/consolidate version

### LINKED ISSUE #3335: Move Suite version to single place?
During every release we are bumping the packge versions to format YY.MM.PATCH (see [docs](https://docs.trezor.io/trezor-suite/misc/versioning.html)). It is a bit annoying that we have to modify the version of each and every packge, see c2448b3a. Could we have just a single file where the version is stored? Or extrapolate it somehow to a single variable? I am not sure if it is possible the way yarn works.

We could also write a tool/regex for that but that seems like an overkill.

### PR DESCRIPTION
**Pull request by goodhoko**
fix #3335 

What to test
- CI produces valid builds.

### CODE REVIEW

**Comment by goodhoko** [comment]
@tsusanka I'd need to run the desktop build jobs for this branch to check they produce artifacts with appropriate versions. If I'm not mistaken I currently don't have the privileges to invoke CI jobs at GitLab. Can I get them?

**Comment by tsusanka** [comment]
Sure, sent vi Slack. Your CI is failing beforehand btw.

File: ci/packages/suite-desktop.yml
Comment (vdovhanych) [inline ci/packages/suite-desktop.yml]: This won't work as the jq utility is only on deployment runner in nix-shell for this to work I will need to add it back to the base docker image. Also, why do we need to get the suite version here?

File: ci/packages/suite-desktop.yml
Comment (vdovhanych) [inline ci/packages/suite-desktop.yml:34]: Same issue

**Review by vdovhanych — COMMENTED** [review]
The ci is failing most likely because of missing utilities on the runner and inside base docker container.

File: ci/packages/suite-desktop.yml
Comment (vdovhanych) [reply ci/packages/suite-desktop.yml]: Nvm, I forgot to get rid of it so it's present in the base docker image. this should work with the current base image sha from develop.

Comment (goodhoko) [reply ci/packages/suite-desktop.yml]: The `SUITE_VERSION` variable is used by the yarn task one line below.

https://github.com/trezor/trezor-suite/blob/430a489e24629c0b3944373e9d5c11fae5c20550/packages/suite-desktop/package.json#L18-L20

I agree this is opaque. We technically could pass the `--c.extraMetadata.version` here in the CI manifest like so
```yaml
     - yarn workspace @trezor/suite-desktop build:${platform} --c.extraMetadata.version=$SUITE_VERSION
```

I decided not to because I wanted the `yarn build:xxx` task to fail loudly (eg. when run locally without the `SUITE_VERSION` veriable set) instead of silently using the wrong version from `packages/suite-desktop/package.json`

I added an explaining comment in 430a489e24629c0b3944373e9d5c11fae5c20550 hope that makes it clearer.

**Comment by goodhoko** [comment]
The CI currently fails on typechecking problems that are caused by the base branch (`feat/tschuss-next`) which is being worked on in [another PR](https://github.com/trezor/trezor-suite/pull/3446).

I'll put this on hold until that PR is merged.

File: ci/packages/suite-desktop.yml
Comment (matejkriz) [reply ci/packages/suite-desktop.yml]: I rewrote it directly to npm scripts using node to deal with missing `jq` packages on multiple environments

File: packages/suite/package.json
Comment (szymonlesisz) [inline packages/suite/package.json:4]: i would say that this should be the source of truth for __suite__ versions, not the package.json in root

Comment (matejkriz) [reply packages/suite/package.json:4]: moved from top level package.json to `suiteVersion` custom param of `packages/suite/package.json`
5d8768269dff7a0d991db7633a12502b6393f14f

**Review by matejkriz — APPROVED** [review]

File: docs/releases/versioning.md
Comment (tomasklim) [inline docs/releases/versioning.md]:
```suggestion
That is, all packages that have `private: true` in their `package.json` and are not consumed by third parties nor published to NPM. Because they get only consumed by other packages in this repo (eg. `@trezor/suite-data` or `@trezor/suite`) by the Yarn's [workspace resolution](https://classic.yarnpkg.com/en/docs/workspaces/) or are distributed in other forms like, for example, bundled applications (eg. `@trezor/suite-desktop`) or websites (eg. `@trezor/suite-web-landing`) we do not version them. That is, their version is kept at `1.0.0` all the time.
```
`@trezor/landing-page` does not exist anymore, right?

File: packages/suite-build/configs/base.webpack.config.ts
Comment (tomasklim) [inline packages/suite-build/configs/base.webpack.config.ts]:
```suggestion
// Get Suite App version from the Suite package.json
```

**Review by tomasklim — CHANGES_REQUESTED** [review]

**Review by tomasklim — APPROVED** [review]

File: docs/releases/versioning.md
Comment (matejkriz) [reply docs/releases/versioning.md]: nice catch 👍

**Pull request merged**
